tmt/export/polarion.py

```diff
@@ -219,6 +219,10 @@ def export_to_polarion(test: tmt.base.Test) -> None:
     description = summary
     if test.description:
         description += ' - ' + test.description
+    if test.environment:
+        description += '<br/>Environment variables:'
+        for envvar in test.environment:
+            description += f'<br/>{envvar}={test.environment[envvar]}'
```

Comment:
How about iterating over `items()`? `for key, value ...` would produce the value right away, it will be needed anyway and it'd make the f-string simpler by removing the nested `[]`.